yarelismartin/skill-swap
# Adjusted community page container

## Description
I have adjusted every page to make it responsive to devices other than a desktop. 

## Motivation and Context
Wanted users to be able to use this application on their mobile devices. 
## How Can This Be Tested?
You can go to each page in the app and make sure that the styling isn't affecting the user experience. 
## Screenshots (if appropriate):

## Types of changes

[ ] Bug fix (non-breaking change which fixes an issue)
x[ ] New feature (non-breaking change which adds functionality)
[ ] Breaking change
